inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/detection_output.cpp

```diff
@@ -61,15 +138,84 @@ INSTANTIATE_TEST_SUITE_P(smoke_DetectionOutput3In, DetectionOutputLayerTest, par
 /* =============== 5 inputs cases =============== */
 
 const std::vector<ParamsWhichSizeDepends> specificParams5In = {
-    ParamsWhichSizeDepends{true, true, true, 1, 1, {1, 60}, {1, 165}, {1, 1, 60}, {1, 30}, {1, 60}},
-    ParamsWhichSizeDepends{true, false, true, 1, 1, {1, 660}, {1, 165}, {1, 1, 60}, {1, 30}, {1, 660}},
-    ParamsWhichSizeDepends{false, true, true, 1, 1, {1, 60}, {1, 165}, {1, 2, 60}, {1, 30}, {1, 60}},
-    ParamsWhichSizeDepends{false, false, true, 1, 1, {1, 660}, {1, 165}, {1, 2, 60}, {1, 30}, {1, 660}},
-
-    ParamsWhichSizeDepends{true, true, false, 10, 10, {1, 60}, {1, 165}, {1, 1, 75}, {1, 30}, {1, 60}},
-    ParamsWhichSizeDepends{true, false, false, 10, 10, {1, 660}, {1, 165}, {1, 1, 75}, {1, 30}, {1, 660}},
-    ParamsWhichSizeDepends{false, true, false, 10, 10, {1, 60}, {1, 165}, {1, 2, 75}, {1, 30}, {1, 60}},
-    ParamsWhichSizeDepends{false, false, false, 10, 10, {1, 660}, {1, 165}, {1, 2, 75}, {1, 30}, {1, 660}}
+    // dynamic input shapes
+    ParamsWhichSizeDepends {
+        true, true, true, 1, 1,
+        {{ov::Dimension::dynamic(), ov::Dimension::dynamic()}, {{1, 60}}},
+        {{ov::Dimension::dynamic(), ov::Dimension::dynamic()}, {{1, 165}}},
+        {{ov::Dimension::dynamic(), ov::Dimension::dynamic(), ov::Dimension::dynamic()}, {{1, 1, 60}}},
+        {{ov::Dimension::dynamic(), ov::Dimension::dynamic()}, {{1, 30}}},
+        {{ov::Dimension::dynamic(), ov::Dimension::dynamic()}, {{1, 60}}},
```

Comment:
We need more than one target shapes, to tests dynamic execution (few infer requests with different target shapes). Also we need test instances with dynamic shapes with defined upper bound.

Reply:
done, thanks